css/geometry/META.yml

```
@@ -18,3 +18,7 @@ links:
           subtest: 'DOMRect interface: legacy window alias'
         - test: idlharness.any.html
           subtest: 'DOMMatrix interface: legacy window alias'
+    - product: chrome
+      url: https://crbug.com/1141729
```

```suggestion
      url: https://crbug.com/709001
```

I found https://bugs.chromium.org/p/chromium/issues/detail?id=709001 and duped 1141729 into that.